lib/colorls/core.rb

```diff
@@ -205,7 +205,11 @@ def group_info(stat)
 
     def size_info(stat)
       size = Filesize.from("#{stat.size} B").pretty.split(' ')
-      "#{size[0][0..-4].rjust(3,' ')} #{size[1].ljust(3,' ')}".colorize(@colors[:normal])
+      size = "#{size[0][0..-4].rjust(3,' ')} #{size[1].ljust(3,' ')}"
+      size = size.colorize(@colors[:file_small])
+      size = size.colorize(@colors[:file_medium]) if stat.size >= 128 * 1024 ** 2
+      size = size.colorize(@colors[:file_large])  if stat.size >= 512 * 1024 ** 2
+      size
```

Comment:
I think lines 209 - 212 can be rewritten as shown below, to be more readable maybe?

```ruby
return size.colorize(@colors[:file_large])  if stat.size >= 512 * 1024 ** 2
return size.colorize(@colors[:file_medium]) if stat.size >= 128 * 1024 ** 2
size.colorize(@colors[:file_small])
```

Reply:
True, I haven't used ruby that much so not aware of the syntactic sugar. Will change!